src/qibolab/instruments/simulator/emulator_test/ibmfakebelem_q01/ibmfakebelem_q01_runcard_dt.yml

```diff
@@ -0,0 +1,193 @@
+nqubits: 5
```

Comment:
This is also duplicating parameters.json and should be removed.

Reply:
Fixed in e035cb3 and 297f8f8